mk/arm.mk

```diff
@@ -9,8 +9,8 @@ export ARM_EXEC
 
 arm-specific-defs = \
     $(Q)$(PRINTF) \
-" /* target: ARM */\n\
-  \#define ARCH_PREDEFINED \"__arm__\" /* defined by GNU C and RealView */\n\
-  \#define ELF_MACHINE 0x28 /* up to ARMv7/Aarch32 */\n\
-  \#define ELF_FLAGS 0x5000200\n\
-"
+        "/* target: ARM */\n$\
```

Comment:
Don't do this! Minimize the necessary changes.

Reply:
This makes the generated configure file aligned left. Note there's a whitespace at the beginning of lines before.
```diff
- /* target: ARM */
- #define ARCH_PREDEFINED "__arm__" /* defined by GNU C and RealView */
- #define ELF_MACHINE 0x28 /* up to ARMv7/Aarch32 */
- #define ELF_FLAGS 0x5000200
+/* target: ARM */
+#define ARCH_PREDEFINED "__arm__" /* defined by GNU C and RealView */
+#define ELF_MACHINE 0x28 /* up to ARMv7/Aarch32 */
+#define ELF_FLAGS 0x5000200
```
Replacing `$\` with `\` and rearranging the string to eliminate that and keep the content easy to read.

Reply:
> Replacing `$\` with `\` and rearranging the string to eliminate that and keep the content easy to read.

Do that in another pull request.